bug: docker.sh restart() references non-existent Makefile target `docker-dev-logs`

## Description

`scripts/docker.sh` `restart()` function (line 237) tells users to run `make docker-dev-logs` to view logs after restarting Docker services, but this Makefile target does not exist. The correct target is `make docker-logs`.

## Steps to Reproduce

1. Run `make docker-restart` (or `./scripts/docker.sh restart`)
2. Copy the suggested command from the output: `make docker-dev-logs`
3. Run it

## Expected Behavior

The command works and shows Docker logs.

## Actual Behavior

```
make: *** No rule to make target 'docker-dev-logs'.  Stop.
```

## Fix

One-line change: `make docker-dev-logs` → `make docker-logs` in `scripts/docker.sh` line 237.

--- a/scripts/docker.sh
+++ b/scripts/docker.sh
@@ -234,7 +234,7 @@ restart() {
     echo -e "${GREEN}✓ Docker services restarted${NC}"
     echo ""
     echo "  🌐 Application: http://localhost:2026"
-    echo "  📋 View logs: make docker-dev-logs"
+    echo "  📋 View logs: make docker-logs"
     echo ""
 }
 

